web/src/test/java/org/openmrs/web/filter/startuperror/StartupErrorFilterTest.java

```diff
@@ -11,41 +11,45 @@
 
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
-import static org.powermock.api.mockito.PowerMockito.mockStatic;
 
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openmrs.test.jupiter.BaseContextSensitiveTest;
 import org.openmrs.web.Listener;
-import org.powermock.core.classloader.annotations.PowerMockIgnore;
-import org.powermock.core.classloader.annotations.PrepareForTest;
-import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.reflect.Whitebox;
 import org.springframework.mock.web.MockHttpServletRequest;
 
 /**
  * Tests {@link StartupErrorFilter}.
  */
-@RunWith(PowerMockRunner.class)
-@PrepareForTest(Listener.class)
-@PowerMockIgnore({"com.sun.org.apache.xerces.*", "javax.xml.*", "org.xml.*", "javax.management.*", "org.w3c.dom.*"})
-public class StartupErrorFilterTest {
+public class StartupErrorFilterTest extends BaseContextSensitiveTest {
+
+	private StartupErrorFilter filter;
 	
-	@Before
+	private Throwable errorAtStartup ;
+
+	@BeforeEach
 	public void setUp() {
-		mockStatic(Listener.class);
+		filter = new StartupErrorFilter();
+	}
+	
+	@AfterEach
+	public void reverterrorAtStartup() {
+		errorAtStartup = null;
+		Whitebox.setInternalState(Listener.class, "errorAtStartup", errorAtStartup);
+		
 	}
 	
 	@Test
 	public void getModel_shouldReturnAStartupErrorFilterModelContainingTheStartupError() {
 		
 		Throwable t = mock(Throwable.class);
-		when(Listener.getErrorAtStartup()).thenReturn(t);
+		Whitebox.setInternalState(Listener.class, "errorAtStartup", t);
```

Comment:
I think we don't need to mock the throwable. You can just create any `Exception` as they are `Throwable`